### ISSUE TITLE
NameError: name '_raise_glfw_errors_as_exceptions' is not defined

### ISSUE THREAD

**Issue by aravindr93** [FlorianRhiem/pyGLFW#44]
```
In __init__.py, line 840, in set_error_callback
    if previous_callback is not None and previous_callback[0] != _raise_glfw_errors_as_exceptions:
NameError: name '_raise_glfw_errors_as_exceptions' is not defined
```
For my use case, downgrading to version 1.9 solved it. Leaving it open here to address, so that projects that have pyglfw as a requirement (e.g. mujoco_py) are not affected downstream.

**Comment by FlorianRhiem** [comment]
Thank you for reporting this, Aravind! I must have missed these two uses of the old default error callback. I'll fix it right away.

**Comment by FlorianRhiem** [comment]
I've fixed the issue in 248af9f8ea7eca7960fcdde1fb76c83ecd3fea3d and released it as [v1.10.1](https://github.com/FlorianRhiem/pyGLFW/releases/tag/v1.10.1).
